com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -3,6 +3,7 @@
 using UnityEngine;
 using System.Linq;
 using Unity.Collections;
+using UnityEditor.PackageManager;
```

Comment:
we can't ship editor code to runtime.

Reply:
oh no... I didn't realize that was editor only... 😞